editor/scene/sprite_frames_editor_plugin.cpp

```diff
@@ -2239,6 +2234,11 @@ SpriteFramesEditor::SpriteFramesEditor() {
 	move_up->connect(SceneStringName(pressed), callable_mp(this, &SpriteFramesEditor::_up_pressed));
 	move_down->connect(SceneStringName(pressed), callable_mp(this, &SpriteFramesEditor::_down_pressed));
 
+	play_bw_from->set_shortcut(ED_SHORTCUT("sprite_frames/play_animation_backwards", TTRC("Play selected animation backwards from current pos."), Key::A));
```

Comment:
You can't just move tooltips to shortcut name, they are whole sentences. Shortcut names should be names of actions., e..g "Play Backwards".
Restore the tooltips, but remove the bracket part, as shortcuts are automatically shown in tooltip.

Reply:
Yeah, that's inconsistent with other shortcuts. They don't end with a period though, at least.